src/docs/content/builders/dialog.md

```diff
@@ -143,7 +143,7 @@ Dialogs can be nested. For example, here's a dialog that opens another dialog.
 ## Accessibility
 
 Adheres to the
-[Dialog WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) &
+[Dialog WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/) and
```

Comment:
We can keep the &s. Makes it quicker to read

Reply:
It's non-standard and inconsistent.

Reply:
I don't mind 🙂